js/src/gallery-block/save.js

```diff
@@ -1,28 +1,5 @@
-/**
- * Internal dependencies
- */
-import { setupAttributesForRendering, sortObject } from './utils';
-
-const Save = ( { attributes } ) => {
-	let configString = '';
-
-	if ( attributes.selectedImages.length ) {
-		const { customSettings, ...config } = setupAttributesForRendering(
-			attributes
-		);
-
-		configString = JSON.stringify( {
-			cloudName: CLDN.mloptions.cloud_name,
-			...sortObject( { ...config, ...customSettings } ),
-		} );
-	}
-
-	return (
-		<div
-			className={ attributes.container }
-			data-cloudinary-gallery-config={ configString }
-		></div>
-	);
-};
+const Save = ( { attributes } ) => (
+	<div className={ attributes.container }></div>
```

Comment:
Question: Does the `container` attribute contain the block instance reference that we can use to extract the block-specific attribute overrides?

Reply:
This is mostly needed so that the gallery widget lib has a particular selector to target.

Reply:
Thanks for confirming! Is it unique for each block?

Reply:
Yep, I generate this random class name in edit.js.